Mathlib/Data/Fintype/Basic.lean

```diff
@@ -256,6 +259,10 @@ theorem image_univ_of_surjective [Fintype β] {f : β → α} (hf : Surjective f
   eq_univ_of_forall <| hf.forall.2 fun _ => mem_image_of_mem _ <| mem_univ _
 #align finset.image_univ_of_surjective Finset.image_univ_of_surjective
 
+theorem _root_.Equiv.finset_image_univ_eq_univ [Fintype β] [DecidableEq β]
+    (f : α ≃ β) : univ.image f = univ :=
+  Finset.image_univ_of_surjective f.surjective
```

Comment:
This is almost identical to `Finset.map_univ_equiv` a few lines below. Can you make the names consistent, and perhaps mark this `simp` too?